### ISSUE TITLE
NameError: name 'filter_by' is not defined

### ISSUE THREAD

**Issue by ravik020** [kieferk/dfply#50]
I am not able to call any of the functions in dfply package. The package has been imported and installed. I am using python 3.0 in JupyterLab notebook.

![image](https://user-images.githubusercontent.com/30272987/38148366-d12f18b2-341b-11e8-8e31-3b305fffe9d1.png)

**Comment by sharpe5** [comment]
Use mask() instead.

This is one of the few instances where the Python API is different to the R API. The reason is a clash with an existing Python API in a different package.

**Comment by ravik020** [comment]
No, what i meant to say is no functions are working from the dfply package. for instance,
![image](https://user-images.githubusercontent.com/30272987/38168154-5b54c23c-350a-11e8-9f52-781742a099d3.png)

**Comment by sharpe5** [comment]
I have been using dfply for years, and it works perfectly every time.

It looks like you are using the wrong syntax. Try some of the examples in the dfply readme at:
https://github.com/kieferk/dfply

Example (diamonds is a built in dataframe):
diamonds >> select(1, X.price, ['x', 'y']) >> head(2)

**Comment by sharpe5** [comment]
Can you also try updating from Python 3.0 to Python 3.6.2? This is the version of Python that I am using. This may or may not make a difference.

**Comment by sharpe5** [comment]
Also, try this, which is probably the simplest test of dfply:

from dfply import *
diamonds >> head(3)

**Comment by kieferk** [comment]
Years! Has it really been around that long? Crazy.

If none of the functions work for you some other strange issue is going on. And for the record, yes as @sharpe5 says `filter` is a reserved name in python and so I switched the name to mask.

**Comment by sharpe5** [comment]
Can we close this issue for now?

**Comment by ravik020** [comment]
Yes
